### PR TITLE
Python Program to Merge two arrays without extra space

### PR DESCRIPTION
**Pull request by narayan2111**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Given two sorted arrays arr1[] and arr2[] in non-decreasing order with size n and m. The task is to merge the two sorted arrays in place, i. e., we need to consider all n + m elements in sorted order, then we need to put first n elements of these sorted in the first array and remaining m elements in the second array.

Sample I/O:

    Input:

    Enter size of 1st array:3
    Enter elements of array 1:
    1 2 3
    Enter size of 2nd array: 2
    Enter elements of array 2:
    1 2

    Output:

    Array after merge:
    1 1 2 2 3

Time Complexity: O(NlogN)(where N will be max of size1 and size2)
Space Complexity: O(1)

### Checklist

- [ Y] I've read the contribution guidelines.
- [ Y] I've checked the issue list before deciding what to submit.
- [ Y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#909

### CODE REVIEW

**Comment by narayan2111** [comment]
Kindly review this PR @iamrajiv @Kajol-Kumari @plazzy99 @Abhijit2505

File: Python/README.md [hunk 1]
```diff
@@ -13,6 +13,16 @@
 
 _add list here_
 
+## Other
+* [Check Inversion](other/Check_Inversion.py)
+* [Kadane's Algorithm](other/Kadanes_Algorithm.py)
+* [Merge two arrays](other/merge_sorted_arrays.py)
+* [Reverse a string using stack](other/Reverse_string_using_stack.py)
```
Comment (atarax665) [inline Python/README.md:20]: Manually edit the readme file to link your program, don't make changes to any existing links.

**Review by atarax665 — DISMISSED** [review]

File: Python/README.md (hunk 1 above)
Comment (narayan2111) [reply Python/README.md:20]: I added it manually, actually there were so many repeated files, i tally them with TesseractCoding master branch and remove those files which were not present in the directory but added in the readme file.

**Comment by narayan2111** [comment]
But i didn't understood, what changes I need to make sir? @atarax665

**Comment by narayan2111** [comment]
> But i didn't understood, what changes I need to make sir? 
Guide me through this. @atarax665

File: Python/README.md (hunk 1 above)
Comment (atarax665) [reply Python/README.md:20]: Revert the changes and just add your program link in the end of the Other heading by editing the README.md file.

Comment (narayan2111) [reply Python/README.md:20]: > Revert the changes and just add your program link in the end of the Other heading by editing the README.md file.

But sir, if i do so, then readme file will have repetitive blocks of read me file in itself. Before making changes into readme file, i saw there were so repetitive blocks of files.

Also before making this Branch, i updated my master branch with TesseractCoding master branch to have recent changes.

**Review by vatsalkesarwani12 — APPROVED** [review]

**Review by iamrajiv — APPROVED** [review]

File: Python/README.md (hunk 1 above)
Comment (atarax665) [reply Python/README.md:20]: I have formatted README.md. Update your branch and add the link of your program under appropriate heading.

Comment (narayan2111) [reply Python/README.md:20]: > I have formatted README.md. Update your branch and add the link of your program under appropriate heading.

Done Sir @atarax665

**Comment by narayan2111** [comment]
Kindly review this PR @Kajol-Kumari

**Review by Kajol-Kumari — APPROVED** [review]
Code LGTM :+1: Just a minor suggestion, Insted of leaving two line gap,please try to leave a single line gap wherever required.

**Comment by narayan2111** [comment]
> Code LGTM 👍 Just a minor suggestion, Insted of leaving two line gap,please try to leave a single line gap wherever required.

okay ma'am

**Comment by narayan2111** [comment]
> Code LGTM 👍 Just a minor suggestion, Insted of leaving two line gap,please try to leave a single line gap wherever required.

Done Ma'am. kindly merge this PR. it has 3 reviews. :)

**Comment by narayan2111** [comment]
kindly merge this PR sir @iamrajiv

**Pull request merged**